satellite/metabase/rangedloop/service.go

```diff
@@ -19,28 +20,39 @@ var (
 )
 
 // Config contains configurable values for the shared loop.
-type Config struct{}
+type Config struct {
+	Parellelism        int           `help:"how many chunks of segments to process in parallel" default:"1"`
+	BatchSize          int           `help:"how many items to query in a batch" default:"2500"`
+	AsOfSystemInterval time.Duration `help:"as of system interval" releaseDefault:"-5m" devDefault:"-1us" testDefault:"-1us"`
```

Comment:
Could we call it like in the other configs in metabase,ie. "AsOfSystemTime"?

Reply:
If I grep for `AsOfSystem(.*)time.Duration` this is the only convention I can find in metabase.